test_unstructured/documents/test_elements.py

```diff
@@ -82,7 +78,7 @@ def bad_cleaner(s: str):
     text_element = Text(text="[1] A Textbook on Crocodile Habitats")
 
     with pytest.raises(ValueError, match="Cleaner produced a non-string output."):
-        text_element.apply(bad_cleaner)  # pyright: ignore[reportGeneralTypeIssues]
+        text_element.apply(bad_cleaner)  # pyright: ignore[reportArgumentType]
```

Comment:
Looks like the latest version of `pyright` defined some more refined type-error types to use instead of `GeneralTypeIssue`. These narrow the ignore to the more descriptive error actually encountered. In this case we're purposely using an argument of the wrong type to trigger the exception behavior under test.